model/otlpgrpc/logs.go

```diff
@@ -72,6 +81,14 @@ func (lr LogsResponse) MarshalJSON() ([]byte, error) {
 	return buf.Bytes(), nil
 }
 
+// UnmarshalJSON unmarshalls LogsResponse from JSON bytes.
+func (lr LogsResponse) UnmarshalJSON(data []byte) error {
```

Comment:
Should we add a `var _ encoding.InterfaceName = (*LogsResponse)(nil)` interface assertion to document what interface we are implementing?

Reply:
Done.